wot-wg-2023-draft.html

```diff
@@ -437,34 +437,33 @@ <h2>Success Criteria</h2>
-
-        <section>
-          <h3 id="w3c-coordination">W3C Groups</h3>
-          <dl>
+  <section id="coordination">
+    <h2>Coordination</h2>
+    <p>For all specifications, this Working Group will seek <a href="https://www.w3.org/Guide/documentreview/#how_to_get_horizontal_review">horizontal review</a> for
+      accessibility, internationalization, privacy, and security with the relevant Working and
+      Interest Groups, and with the <a href="https://www.w3.org/2001/tag/" title="Technical Architecture Group">TAG</a>.
+      Invitation for review must be issued during each major standards-track document transition, including
+      <a href="https://www.w3.org/Consortium/Process/#RecsWD" title="First Public Working Draft">FPWD</a>.  The
+      Working Group is encouraged to engage collaboratively with the horizontal review groups throughout development of
+      each specification.  The Working Group is advised to seek a review at least 3 months before first entering
+      <a href="https://www.w3.org/Consortium/Process/#RecsCR" title="Candidate Recommendation">CR</a> and is encouraged
+      to proactively notify the horizontal review groups when major changes occur in a specification following a review.</p>
+
+    <p>Additional technical coordination with the following Groups will be made, per the <a href="https://www.w3.org/Consortium/Process/#WGCharter">W3C Process Document</a>:</p>
+
+    <p class="todo">In addition to the above catch-all reference to horizontal review which includes accessibility review, please check with chairs and staff contacts of the <a href="https://www.w3.org/WAI/APA/">Accessible Platform Architectures Working Group</a> to determine if an additional liaison statement with more specific information about concrete review issues is needed in the list below.</p>
+
+  <section>
+    <h3 id="w3c-coordination">W3C Groups</h3>
+    <dl>
 <!--
             <dt><a href="https://w3c.github.io/charter-drafts/charter-template.html"><i class="todo">[other name]</i> Working Group</a></dt>
               <dd><i class="todo">[specific nature of liaison]</i></dd>
 -->
-            <dt><a href="https://www.w3.org/community/wot/">Web of Things Community Group</a></dt>
-              <dd>For collaboration on community outreach of the WG reports to increase adoption and implementation, 
-		  as well as collecting feedback from the global community. 
-		  Meetings held in English.
-		  </dd>	
+    <dt><a href="https://www.w3.org/community/wot/">Web of Things Community Group</a></dt>
+      <dd>For collaboration on community outreach to increase adoption, implementation, and understanding, as well as 
+        collecting feedback from the global community. Meetings held in English.
+    </dd>	
             <dt><a href="https://www.w3.org/community/wot-jp/">Web of Things Japanese Community Group</a></dt>
               <dd>For collaboration on community outreach of the WG reports to increase adoption and implementation, 
```

Comment:
To make things consistent...
```suggestion
              <dd>For collaboration on community outreach to increase adoption and implementation, 
```